contracts/modules/SuperMinterV1_1.sol

```diff
@@ -285,16 +307,13 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
             d.maxMintable = c.maxMintable;
             d.affiliateFeeBPS = c.affiliateFeeBPS;
             d.mode = c.mode;
-            d.flags = _MINT_CREATED_FLAG | LibOps.toFlag(usePlatformSigner, _USE_PLATFORM_SIGNER_FLAG);
+            d.flags = _MINT_CREATED_FLAG;
```

Comment:
Sounds like everything is going to use the platform signer. Is that correct?

Reply:
that's correct